### ISSUE TITLE
[Feature] Moving around files in the webinterface

### ISSUE THREAD

**Issue by UnlimitedCookies** [go-gitea/gitea#11153]
Hi! It would be awesome if we could move around files and folder from within the web interface. This would be even better with drag and drop support.

This is a good feature for people that are not familiar in working with git software and prefer to use the web interface for such operations.
Please let me know what you think about this request.

**Comment by zeripath** [comment]
You already can move files around. Just edit the file and change its filename - press backspace at the start of the filename input field to remove subdirectories.

**Comment by zeripath** [comment]
I'll keep this open in case I've misunderstood what you were asking for though and for the drag-n-drop feature

**Comment by UnlimitedCookies** [comment]
You're right about that. But unfortunately this only applies to text files that are editable with the web interface editor.
Binaries or images cannot be moved with the web interface.
And drag-n-drop would of course be a nice to have feature.

**Comment by UnlimitedCookies** [comment]
The following bullet points are still open (refer my answer above): 
- Only files that can be edited with the gitea text editor are moveable from within the web interface
- Moving is done by pressing backspace (not apparent/beginner-friendly)
- Drag and Drop would be pretty nice

**Comment by xuan-nguyen-swe** [comment]
I write books, docs, blogs using markdown with the help of tools like Docusaurus.
For these repositories, I prefer to use gitea web ui to edit markdown files directly.
This way is so portable and convenient for the purpose above.
I don't need to work on my PC, don't need to open an IDE or text editor.
I can preview then the integrated CI tool can deploy the project immediately.

However, moving multiple files and folder is painful for this type of projects.
It would be nice to have a simple but user friendly to move multiple files and folders quickly via the web ui.

**Comment by davama** [comment]
> You already can move files around. Just edit the file and change its filename - press backspace at the start of the filename input field to remove subdirectories.

this is not the case for non-text files.

yes, drag and drop would be a nice have. or at least ability to edit the filename/path seperately

**Comment by wxiaoguang** [comment]
> this is not the case for non-text files.

Binary file is also supported now
